packages/widged/.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
```

Comment:
Should this package be called "widge**d**"? What about "widge**t**"

Reply:
I was trying to be creative here so I called it Widged from WIDGet and EDitor 😅